srtcore/buffer_rcv.cpp

```diff
@@ -1050,13 +1050,13 @@ void CRcvBuffer::updateTsbPdTimeBase(uint32_t usPktTimestamp)
 
 string CRcvBuffer::strFullnessState(bool enable_debug_log, int iFirstUnackSeqNo, const time_point& tsNow) const
 {
+    if (!enable_debug_log)
```

Comment:
The reason of this `enable_debug_log` check:
https://github.com/Haivision/srt/pull/2463#discussion_r981061159
If return empty string, why not just check `enable_debug_log` in the caller site?

Reply:
Because the idea is to avoid the lengthy evaluation when the resulting string will be ignored anyway. The instruction for logging will not result in a printed log if the log isn't enabled, but all the arguments will be evaluated anyway.

Reply:
> the lengthy evaluation

Does it mean `qrlog.Debug.CheckEnabled() ? strFullnessState() : ""` ?

Reply:
Probably would work as well. @maxsharabayko what do you think?

Reply:
> Does it mean `qrlog.Debug.CheckEnabled() ? strFullnessState() : ""` ?

Good suggestion. And get rid of the `enable_debug_log` argument.